src/main/services/sealedClaimPdfGenerator.ts

```diff
@@ -16,27 +16,12 @@ export class SealedClaimPdfGenerator {
    *
    * @param {e.Request} req HTTP request
    * @param {e.Response} res HTTP response
-   * @param {e.NextFunction} next next handler
    * @returns {Promise<void>}
    */
-  static async requestHandler (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> {
+  static async requestHandler (req: express.Request, res: express.Response): Promise<void> {
     const claim: Claim = res.locals.claim
 
-    try {
-      documentsClient.getSealedClaimPDF(claim.externalId, res.locals.user.bearerToken)
-        .on('response', (response: http.IncomingMessage) => {
-          try {
-            pdfEndpointResponseHandler(`sealed-claim-${claim.claimNumber}`, res)(response)
-          } catch (err) {
-            next(err)
-          }
-        })
-        .on('error', (err: Error) => {
-          next(err)
-        })
-    } catch (error) {
-      next(error)
-    }
-
+    const pdf: Buffer = await documentsClient.getSealedClaimPDF(claim.externalId, res.locals.user.bearerToken)
+    DownloadUtils.downloadPDF(res, pdf, `sealed-claim-${claim.claimNumber}`)
```

Comment:
Shall I change file name to `{claim.claimNumber}-sealed-claim` for consistency?

Reply:
i would say so, I honestly don't think defendant's care about `sealed` though

Reply:
Renamed